canonical-websites/www.ubuntu.com
# Deep link all redirects to community.ubuntu.com

## Done
Deep linked all redirects to community.ubuntu.com. Added countdown and bounties to the deleted list.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the redirects to community.ubuntu.com go to the right places.

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2528
